src/Organisms/EnhancedList/EditColumns/EditColumns.jsx

```diff
@@ -34,24 +34,26 @@ const EditColumns = ({
   const disabledColumns = columns.filter(({ show }) => !show)
```

Comment:
Looping over the columns multiple times like that is quite inneficient, that would be a good use of Array.reduce():
```js
const { enabledColumns, disabledColumns } = columns.reduce(
  (groups, column) => {
    groups[column.show ? 'enabledColumns' : 'disabledColumns'].push(column)
    return groups
  },
  { enabledColumns: [], disabledColumns: [] }
)
```